app/grandchallenge/retina_api/urls.py

```diff
@@ -1,5 +1,5 @@
 from django.urls import path, include
-from rest_framework.routers import DefaultRouter
+from rest_framework.routers import DefaultRouter, SimpleRouter
```

Comment:
What URLs does the default router provide now? Just the root page as there are no models registered with it?

Reply:
Yes, which is not used. I removed it in df95f72566d0b01be27342f0da42980d3d8a1b60